### PR TITLE
Add TikTok icon in social buttons

### LINKED ISSUE #1766: Add TikTok icon in social buttons
TikTok social icon has been added to the UI Kit v5.0.

So this issue is very simple: let's add this icon in our documentation as a `.btn-tiktok` in:
* [Social buttons](https://boosted.orange.com/docs/components/buttons/#social-buttons)
* [Dark variant](https://boosted.orange.com/docs/components/buttons/#dark-variant)

Use [svgo](https://www.npmjs.com/package/svgo) to clean svg and then replace all `"` by `'`, like explained in #989.

### PR DESCRIPTION
**Pull request by hannahiss**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

Fixes #1766 

### Description

- add Tiktok icon in scoial buttons
- add Tiktok icons in footer examples
- add info callout in footer doc to inform developers that other social icons are available in social buttons

### Motivation & Context

Missing TikTok icon after DSM V5 release

### Types of change

- New feature (non-breaking change which adds functionality)

### Live previews

- <https://deploy-preview-1778--boosted.netlify.app/docs/components/buttons/#social-buttons>

#### Contribution

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

#### Accessibility

- [x] My change follows accessibility good practices; I have at least run axe

#### Design

- [x] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
- [x] My change is compatible with responsive display

#### Development

- [x] My change follows the [developer guide](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Developer-guide)
- (NA) I have added JavaScript unit tests to cover my changes

#### Documentation

- [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [x] My change introduces changes to the migration guide
- (NA) My new component is added in Storybook
- (NA) My new component is compatible with RTL
- (NA) Manually run [BrowserStack tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
- (NA) Manually test browser compatibility with BrowserStack (Chrome >= 60, Firefox >= 60 (+ ESR), Edge, Safari >= 12, iOS Safari, Chrome & Firefox on Android)
- [ ] Code review
- [ ] Design review
- [ ] A11y review

#### After the merge

- [ ] Manually launch [Percy tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/percy.yml)

### CODE REVIEW

**Comment by hannahiss** [comment]
Waiting for new icons from Franco (Messenger, Discord, ...?)

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
I think the bundle might be a bit too heavy with Solaris icons even if it looks better right now, we should see if we can have some lighter icons ?

File: site/content/docs/5.3/migration.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
 - **Buttons**
   - <span class="badge bg-success">New</span> Outlined button `.btn-outline-secondary` is now provided. It is basically `.btn-secondary` with default transparent background.
+  - <span class="badge bg-success">New</span> TikTok social button
```
Comment (louismaximepiton) [inline site/content/docs/5.3/migration.md:69]:
```suggestion
  - <span class="badge bg-success">New</span> TikTok social button.
```

File: site/content/docs/5.3/components/footer.md [hunk 2]
```diff
@@ -54,7 +54,11 @@ This footer bar is a container with a `.footer-title-content` class. This contai
 
 ### Follow us bar
 
-This footer bar is a container with `.footer-social` class into which a list of social media icons can be added:
+This footer bar is a container with `.footer-social` class into which a list of social media icons can be added.
+
+{{< callout info >}}
+Other social icons are available. See full list in [Social buttons section]({{< docsref "/components/buttons#social-buttons" >}}).
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/footer.md:60]:
```suggestion
See the full list of our provided socials in our [Social buttons section]({{< docsref "/components/buttons#social-buttons" >}}).
```

**Review by louismaximepiton — APPROVED** [review]
Some wording proposals, but already seems fine to me.

File: site/layouts/shortcodes/orange-footer.html [hunk 3]
```diff
@@ -53,6 +53,7 @@ <h3 class="footer-heading me-md-3">Follow us</h3>
         <li><a href="#" class="btn btn-icon btn-social btn-snapchat btn-inverse"><span class="visually-hidden">Snapchat</span></a></li>
         <li><a href="#" class="btn btn-icon btn-social btn-pinterest btn-inverse"><span class="visually-hidden">Pinterest</span></a></li>
         <li><a href="#" class="btn btn-icon btn-social btn-mail btn-inverse"><span class="visually-hidden">Mail</span></a></li>
+        <li><a href="#" class="btn btn-icon btn-social btn-tiktok btn-inverse"><span class="visually-hidden">Tiktok</span></a></li>
```
Comment (julien-deramond) [inline site/layouts/shortcodes/orange-footer.html:56]:
```suggestion
        <li><a href="#" class="btn btn-icon btn-social btn-tiktok btn-inverse"><span class="visually-hidden">TikTok</span></a></li>
```

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**